rllib/utils/replay_buffers/replay_buffer.py

```diff
@@ -141,18 +139,10 @@ def add(self, batch: SampleBatchType, **kwargs) -> None:
             batch: Batch to add to this buffer's storage.
             **kwargs: Forward compatibility kwargs.
         """
-        assert batch.count > 0, batch
-        warn_replay_capacity(item=batch, num_items=self.capacity / batch.count)
+        if not batch.count > 0:
```

Comment:
Cool. Agree! Should be resilient to len==0 batches, which have their justification in some cases.